app/components/publications/publications-table-row.hbs

```diff
@@ -44,97 +86,98 @@ request the whole row and not only the arrow at the end to be clickable }}
-    <td data-test-publication-table-row-number>
-      {{#if @publicationFlow.identification.idName}}
-        {{@publicationFlow.identification.idName}}
+  {{#if @tableColumnDisplayOptions.openingDate}}
+    <td data-test-publication-table-row-opening-date class="auk-u-text-nowrap">
+      {{#if @publicationFlow.openingDate}}
+        {{moment-format @publicationFlow.openingDate "DD-MM-YYYY"}}
       {{else}}
         {{t "dash"}}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.regulationType}}
+  {{#if @tableColumnDisplayOptions.translationRequestDate}}
     <td
-      data-test-publication-table-row-regulation-type
+      data-test-publication-table-row-translation-request-date
       class="auk-u-text-nowrap"
     >
-      {{#if @publicationFlow.regulationType.abbreviation}}
-        {{@publicationFlow.regulationType.abbreviation}}
-      {{else if @publicationFlow.regulationType}}
-        {{@publicationFlow.regulationType.label}}
+      {{!translation-subcase start-date === start-date of first request-activity}}
+      {{#if @publicationFlow.translationSubcase.startDate}}
+        {{moment-format
+          @publicationFlow.translationSubcase.startDate
+          "DD-MM-YYYY"
+        }}
       {{else}}
         {{t "dash"}}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.numacNumber}}
-    <td data-test-publication-table-row-numac-number>
-      {{#if @publicationFlow.numacNumbers.length}}
-        {{join ", " (map-by "idName" @publicationFlow.numacNumbers)}}
+  {{#if @tableColumnDisplayOptions.translationDueDate}}
+    <td
+      data-test-publication-table-row-translation-due-date
+      class="auk-u-text-nowrap"
+    >
+      {{#if @publicationFlow.translationSubcase.dueDate}}
+        {{moment-format
+          @publicationFlow.translationSubcase.dueDate
+          "DD-MM-YYYY"
+        }}
       {{else}}
         {{t "dash"}}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.openingDate}}
-    <td data-test-publication-table-row-opening-date class="auk-u-text-nowrap">
-      {{#if @publicationFlow.openingDate}}
-        {{moment-format @publicationFlow.openingDate "DD-MM-YYYY"}}
+  {{#if @tableColumnDisplayOptions.proofRequestDate}}
+    <td
+      data-test-publication-table-row-proof-request-date
+      class="auk-u-text-nowrap"
+    >
+      {{#if this.proofRequestDate}}
+        {{moment-format
+          this.proofRequestDate
+          "DD-MM-YYYY"
+        }}
       {{else}}
         {{t "dash"}}
       {{/if}}
     </td>
   {{/if}}
-  {{#if @tableColumnDisplayOptions.publicationTargetDate}}
+  {{#if @tableColumnDisplayOptions.proofReceivedDate}}
     <td
-      data-test-publication-table-row-target-end-date
+      data-test-publication-table-row-proof-received-date
       class="auk-u-text-nowrap"
     >
-      {{#if @publicationFlow.publicationSubcase.targetEndDate}}
+      {{#if @publicationFlow.publicationSubcase.receivedDate}}
```

Comment:
Is het zinvol om hier de vroegste datum te nemen? Voor de opvolging van publicatieflows lijkt het me handiger om de datum leeg te laten wanneer er nog een aanvraag open staat.

Reply:
Te bekijken met de gebruikers maar dat zou ik uit deze PR houden, want gaat ook impact hebben op de logica bij het maken van nieuwe aanvragen.